### PR TITLE
Add `tap-msaccess`

### PR DESCRIPTION
**Pull request by ReubenFrankel**
https://github.com/Matatika/tap-msaccess/releases/tag/v0.1.0

New take on the `tap-spreadsheets-anywhere` approach: instead of a single plugin that can connect to multiple file systems, break each out  into its own plugin with defined settings, in an attempt to make configuration easier for the user.

---

Adds 4 new plugins (possible more in the future):

- `tap-msaccess` - the "base" plugin with local file system support only
- `tap-msaccess-azure` - Azure file system support with known settings
- `tap-msaccess-s3` - S3 file system support with known settings
- `tap-msaccess-http` - HTTP file system support
- `tap-msaccess-anywhere` - Support for (theoretically) all [`fsspec`](https://filesystem-spec.readthedocs.io/en/latest/index.html) backend implementations

Nothing set in stone here yet, so if you have any thoughts or concerns, please raise them!

### CODE REVIEW

**Comment by edgarrmondragon** [comment]
> should I set it to `kind: password` for now or ignore assuming that the schema will be updated at some point?

I think so. Meltano is inject the `sensitive: true` flag into lock files anyway: https://github.com/meltano/meltano/pull/7874/files#diff-0a49a7859d1fe309fa7e7f789e398c35653bf50c9c18c82e9d7334cd54c0e5feR229

**Comment by ReubenFrankel** [comment]
Looks like there is some similar discussion in #1445 for `tap-universal-file`.

**Comment by edgarrmondragon** [comment]
[tap-dbf](https://hub.meltano.com/extractors/tap-dbf#s3-endpoint_url-setting) uses the approach of nested settings, e.g. `s3.endpoint_url`. That solves the settings problem, but not the packaging _extras_ problem, though ...

File: _data/default_variants.yml [hunk 1]
```diff
@@ -321,6 +321,10 @@ extractors:
   tap-moosend: hotgluexyz
   tap-ms-graph: slalom-consulting
   tap-ms-teams: singer-io
+  tap-msaccess: matatika
+  tap-msaccess-anywhere: matatika
+  tap-msaccess-azure: matatika
+  tap-msaccess-http: matatika
```
[the remark this answers is not in the document]
Comment (ReubenFrankel) [reply _data/default_variants.yml:327]: Cheers! :sweat_smile:

**Comment by edgarrmondragon** [comment]
This would benefit from https://github.com/meltano/hub/pull/1701

**Review by edgarrmondragon — APPROVED** [review]
Thanks @ReubenFrankel!

**Pull request merged**